specs/tendrl_gdeploy_wrapper.adoc

```diff
@@ -0,0 +1,179 @@
+= Tendrl-Gdeploy wrapper
+
+
+Tendrl should have a wrapper for gdeploy, so that provisioning features
+of gdeploy can be leveraged in tendrl.
+
+== Problem description
+
+gdeploy is an Ansible based deployment tool which can create physical volumes,
+volume groups, and logical volumes, install packages, subscribe to RHN channels,
+run shell commands, create GlusterFS volumes and lot more.
+
+Tendrl as a storage management platfrom will have to perform most of the above
+mentioned tasks. Therefore instead of reinventing the wheel, we can leverage the
+capabilities of gdeploy by having a wrapper on top of it.
+
+This wrapper should have a mechanism to generate the gdeploy config files for the
+tasks that tendrl intends to perform via gdeploy.
+
+== Use Cases
+
+* tendrl-gdeploy wrapper can be used for provisioning gluster nodes
+* It can be used for gluster performing volume management operations
+
+== Proposed change
+=== gdeploy
```

Comment:
Are there any BZs or upstream GDeploy tickets for tracking these changes, please?